Formula/jpeg-xl.rb

```diff
@@ -41,7 +40,7 @@ class JpegXl < Formula
   # https://github.com/libjxl/libjxl/tree/v#{version}/third_party
   resource "highway" do
     url "https://github.com/google/highway.git",
```

Comment:
This project tags commits (and is at 1.0+ now), so it would be good to make this its own formula, I think.